tests/ipfwd/test_nhop_count.py

```diff
@@ -365,9 +371,18 @@ def test_nhop(request, duthost, tbinfo):
     # verify the test used up all the NHOP group resources
     # skip this check on Mellanox as ASIC resources are shared
     if not is_mellanox_device(duthost):
-        pytest_assert(
-            crm_after["available"] == 0,
-            "Unused NHOP group resource: {}, used:{}".format(
-                crm_after["available"], crm_after["used"]
+        if is_cisco_device(duthost):
+            pytest_assert(
+                (crm_after["available"] + nhop_group_count) == crm_before["available"],
```

Comment:
So can I understand it is not a range, it can just fill 92% on Cisco platform as you use "==" here?

Reply:
Yes, we will just reach to 92%.